src/Scripts/Python/PythonModule.cs

```diff
@@ -163,11 +163,17 @@ public override void FireEvent(ScriptEvent @event, Program program)
 
         private static ScriptEngine CreateEngine(TextWriter outputWriter)
         {
+            // Set recursion limit to avoid application crash if there is
+            // infinite recursion
+            var options = new Dictionary<string, object>()
+            {
+                { "RecursionLimit", 100 },
+            };
```

Comment:
Would it make sense to add the `options` dictionary as an optional argument to this method, so that the `RecursionLimit` could be customized? I'm seeing a hard-wired constant `100` which makes me think it might be necessary to allow it to be modified.

Reply:
I have added optional parameter in 32dc546.